src/test/java/CarRecommendationsTest/BudgetFilterTest.java

```diff
@@ -0,0 +1,31 @@
+package CarRecommendationsTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.aviva.CarRecommendations.BudgetFilter;
+import com.aviva.Entities.AccountHolder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BudgetFilterTest {
+    AccountHolder knownUser;
+    BudgetFilter filterToTest;
+
+    @BeforeEach
+    public void setup() {
+        knownUser = new AccountHolder("1402110922112412");
+        filterToTest = new BudgetFilter();
+        knownUser.setSavings(100000F);
+    }
+
+    @Test
+    public void testGetRecommendedCarsOutputSize() {
```

Comment:
maybe rather than just testing size, test that it satisfies the conditions you defined? like <yearlyBudget or whatever it was

Reply:
like in addiiton to this. cuz size is too basic